etc/docs/preview.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from 'express';
+
```

Comment:
The python http server didn't support redirecting endpoints from `node-mongodb-native` to `/`, so I added express as a dev dependency and wrote a simple server to preview docs.